### PR TITLE
osd-11306 ONV boilerplate update

### PR DESCRIPTION
**Pull request by ritmun**
Updated ONV to use convention in boilerplate to handle the build and push of images to the registry instead of the local scripts in 'hack' folder.
 
Jira : https://issues.redhat.com/browse/OSD-11306 
ONV PR dependency: https://github.com/openshift/osd-network-verifier/pull/76

### CODE REVIEW

File: ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml [hunk 1]
```diff
@@ -16,10 +15,6 @@ tests:
   commands: make test
   container:
     from: src
-- as: build
```
Comment (boranx) [inline ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml:19]: if we remove build, how do we test `make build` to make sure verifier binary can be created anyway

Comment (ritmun) [reply ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml:19]: This commit was auto generated by `make prow-config`. I'm not sure what the logic behind removing `make build` is though.

**Comment by ritmun** [comment]
this does not seem to be a requirement any more

**Comment by ritmun** [comment]
this does not seem to be a requirement any more

**Pull request closed without merging**